dashboard/src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -0,0 +1,42 @@
+import { ComponentPropsWithoutRef } from 'react';
+
+import {
+  Breadcrumb,
+  BreadcrumbList as BreadcrumbListComponent,
+  BreadcrumbItem,
+  BreadcrumbLink as BreadcrumbLinkComponent,
+  BreadcrumbPage,
+  BreadcrumbSeparator as BreadcrumbSeparatorComponent,
+  BreadcrumbEllipsis,
+} from '../ui/breadcrumb';
+
+type BreadcrumbListProps = ComponentPropsWithoutRef<'ol'>;
+const BreadcrumbList = ({ ...props }: BreadcrumbListProps): JSX.Element => {
+  return <BreadcrumbListComponent {...props} className="text-black" />;
+};
+
+type BreadcrumbLinkProps = ComponentPropsWithoutRef<'a'> & {
+  asChild?: boolean;
+};
+const BreadcrumbLink = ({ ...props }: BreadcrumbLinkProps): JSX.Element => {
+  return (
+    <BreadcrumbLinkComponent {...props} className="hover:text-slate-500" />
+  );
+};
```

Comment:
Nit: I think it would be clearer and more obvious what's going on if we did that, but it is your call

```suggestion
const BreadcrumbLink = ({ ...props }: typeof BreadcrumbLinkComponent): JSX.Element => {
  return (
    <BreadcrumbLinkComponent {...props} className="hover:text-slate-500" />
  );
};
```

Reply:
improved the type to be
```
type BreadcrumbLinkProps = Omit<
  ComponentProps<typeof BreadcrumbLinkComponent>,
  'className'
>;
```